src/battle/trainer_ai.c

```diff
@@ -0,0 +1,65 @@
+#include "battle/battle.h"
```

Comment:
I gotta say, this doesn't feel like the entire trainer ai file? only a part of it? the way it's named implies that it's the whole thing, unless ofc this is partially decomped (too lazy to check)

Reply:
partially decompiled lol, I started it on request of a friend but quickly realized it wasn't what he was looking for since most of what he needed is in the ai scripts and not the script commands (but I figured documenting it is still good)